### DISCUSSION TITLE
voicegain spanish feature

### DISCUSSION THREAD

**Discussion by johnfelipe** [pietrop/digital-paper-edit-electron#68] [language]
You can test accuracy from Web Console.

First you need to get an account at: Voicegain Cloud Portal: https://console.voicegain.ai/signup

 

Here are some simple instructions how to enable and use the Spanish Recognizer:

Spanish Transcription from Web Console – Voicegain: https://support.voicegain.ai/hc/en-us/articles/360056989172-Spanish-Transcription-from-Web-Console

is possible add this to your great app?

**Comment by pietrop** [comment]
Thanks @johnfelipe great find!

it looks like they have an API https://support.voicegain.ai/hc/en-us/categories/360001288691-Speech-to-Text-API do you know if they have a node SDK?

**Comment by pietrop** [comment]
oh, I found this https://support.voicegain.ai/hc/en-us/articles/360044631632-Example-of-invoking-Voicegain-API-from-node-js-asr-transcribe-

I'd welcome a PR if you want to give it a try?
I can provide some guidance on the steps tho if it helps

**Comment by johnfelipe** [comment]
i want to try, but i have reduced time for do it, IBM watson is a good
feature but with version 2, and i want to use spanish transcriber in
version 3, may be this have a good chance for you roadmap

**Comment by pietrop** [comment]
what other languages does this support?

have you tried IBM Watson in version 2 recently? does it still work?

**Comment by pietrop** [comment]
Also it be great if you could try it in their web console and see if the accuracy if good enough, if you haven't already? before considering adding it to this project

**Comment by johnfelipe** [comment]
Voicegain support english to
Ibm watson still work in version 2, but I hope works in future in version
3..

**Comment by johnfelipe** [comment]
I do counting and after 12 files accuracy is in 94%
Pls add this in roadmap.

**Comment by pietrop** [comment]
ok, I am a bit strapped with time, but let me see what I can do. I think if IBM still works in v2, it might be possible to get it to work in v3. There's a PR here https://github.com/pietrop/digital-paper-edit-electron/pull/52 @seanmtracey was helping figuring out if there's anything on the IBM SDK Side of things that might need fixing

**Comment by johnfelipe** [comment]
Watson in version 2 work perfectly, but version2 is something limited for
edition, pls pls put to work in version3 because is better featured
And about voicegain in spanish is good, obviously watson is better,
voicegain have 600 min monthly and have english supported to.
